sympy/utilities/tests/test_lambdify.py

```diff
@@ -264,7 +264,15 @@ def test_issue_12984():
         warnings.simplefilter("ignore", RuntimeWarning)
         assert str(func_numexpr(-1, 24, 42)) == 'nan'
 
-#================== Test some functions ============================
+
+def test_empty_modules():
+    x, y = symbols('x y')
+    expr = -(x % y)
+
+    no_modules = lambdify([x, y], expr)
+    empty_modules = lambdify([x, y], expr, modules=[])
+    assert no_modules(3, 7) == empty_modules(3, 7)
```

Comment:
What happens for `no_modules(3, 7 + pi)`? It seems like most code printers have an issue printing this...

Reply:
Never mind... It was an issue in another printer, but it was local, so probably it still works here.